### PR TITLE
#704 update landing page

### PR DESCRIPTION
**Pull request by davidemmolino**
Addresses issue #704 
- Installed additional MUI packages to support dropdown and icons for new designs
- bumped types/react version to support MUI packages
- updated and removed unused text
- created an accordions component

**Preview screenshots**
Screen Shot 2022-11-14 at 4 32 19 PM
Screen Shot 2022-11-14 at 4 32 24 PM
Screen Shot 2022-11-14 at 4 32 27 PM
Screen Shot 2022-11-14 at 4 32 32 PM
Screen Shot 2022-11-14 at 4 57 23 PM

**Before** changes
Screen Shot 2022-11-14 at 4 27 41 PM

**After** changes
Screen Shot 2022-11-14 at 4 32 12 PM

### CODE REVIEW

**Comment by davidemmolino** [comment]
Performance is really low, might be due to having a bunch of PNG's on the page. Although, pretty sure Lighthouse is doing it's performance test on the development build of the site. Might be better in prod version of the build.

Not sure if you want to consider lazy loading a landing page or fetching those images from a remote database.

I heard MUI isn't great in terms of performance and that a headless UI could be a better and less bloated option but if we're on a time crunch it's probably best to put the idea on the backburner.

**Review by aahvocado — CHANGES_REQUESTED** [review]
Sweet, thanks for the big chunk of work! I left some comments inline. I'll be sure to respond quicker this time.

I'm going to see if I can ask design for .svg files to reduce that initial load size because these images do indeed have a big hit on performance.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 .ipynb_checkpoints
 venv
 .DS_Store
+node_modules
+.idea/*
```
[the remark this answers is not in the document]
Comment (davidemmolino) [reply .gitignore:5]: done

File: products/statement-generator/package.json [hunk 2]
```diff
@@ -4,15 +4,19 @@
   "homepage": "./",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
     "@material-ui/core": "^4.11.2",
     "@material-ui/icons": "^4.11.2",
+    "@mui/icons-material": "^5.10.9",
+    "@mui/material": "^5.10.13",
```
[the remark this answers is not in the document]
Comment (davidemmolino) [reply products/statement-generator/package.json:12]: icons-material is for the arrow in the accordion, don't necessarily need it.
The rest are needed to generate the MUI accordion itself. 
Are you suggesting just building the accordion from scratch?

File: products/statement-generator/src/components/Accordions.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+// override default MUI styling
+const AccordionSummary = styled((props: AccordionSummaryProps) => (
+  <MuiAccordionSummary
+    expandIcon={<ArrowForwardIosSharpIcon sx={{ fontSize: '0.9rem' }} />}
+    {...props}
+  />
+))(() => ({
+  '& .MuiAccordionSummary-expandIconWrapper.Mui-expanded': {
+    transform: 'rotate(90deg)',
```
[the remark this answers is not in the document]
Comment (davidemmolino) [reply products/statement-generator/src/components/Accordions.tsx:34]: Updated

File: products/statement-generator/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (davidemmolino) [reply products/statement-generator/package.json:12]: Also found this info which I thought was interesting.. Don't usually work with MUI so it was nice to learn
https://mui.com/material-ui/guides/minimizing-bundle-size/

[the remark this answers is not in the document]
Comment (aahvocado) [reply products/statement-generator/package.json:12]: > icons-material is for the arrow in the accordion, don't necessarily need it. The rest are needed to generate the MUI accordion itself. Are you suggesting just building the accordion from scratch?

What I mean is we already have material-ui v4 which has an [accordion component](https://v4.mui.com/components/material-icons/#material-icons) and the [arrows](https://v4.mui.com/components/material-icons/#material-icons) if you look for ExpandMore and ExpandLess. No need to bring in another package with v5

[the remark this answers is not in the document]
Comment (davidemmolino) [reply products/statement-generator/package.json:12]: I see, sorry but both links go to the same url could you update the url for accordion, please?

**Comment by sydneywalcoff** [comment]
Closing this, as it's less work to just rebuild this.

**Pull request closed without merging**